lib/generators/react_on_rails/templates/bootstrap/client/assets/stylesheets/_pre-bootstrap.scss

```diff
@@ -0,0 +1,7 @@
+// The Rails server imports this file via app/assets/stylesheets/application.css.scss
+// The webpack HMR dev server loads this file via client/bootstrap-sass.config.js
+
+// In either case, this file is loaded BEFORE bootstrap has been loaded.
+// Use it to define variables BEFORE bootstrap loads. See http://getbootstrap.com/customize/
+
+@import 'react-on-rails-sass-helper';
```

Comment:
we should explain why this file is required -- for images and fonts

and I think we should define some variable to show it can be done.